lib/types/OktaAuthOptions.ts

```diff
@@ -112,6 +113,12 @@ export interface OktaAuthOptions extends
   storageManager?: StorageManagerOptions;
   services?: ServiceManagerOptions;
   transactionManager?: TransactionManagerOptions;
+  idx?: Pick<IdxTransactionMeta,
+    'flow' |
+    'activationToken' |
+    'recoveryToken' |
```

Comment:
recoveryToken does apply to both idx and classic. 
activationToken (self-service) is new with IDX flow, but it doesn't feel like an IDX option
flow is also new with IDX, but I think it makes sense to keep as a top-level option because it is a first-class feature.
My thought on the "idx" config section is that it is for advanced/internal options that are possibly undocumented and therefore subject to change without triggering a major version release

Reply:
Then how about `idx` -> '_idx' as a internal naming convention?

Reply:
Some customers will be setting these options (with hand-holding, appropriate warnings). But eventually these options will be documented and supported, so they are not "internal", but more like "beta".